modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase

```diff
@@ -865,6 +865,186 @@ definition {
+
+			Button.clickSave();
+		}
+
+		task ("Define workflow to Custom Object") {
+			ObjectAdmin.openWorkflowProcessBuilderConfiguration();
+
+			Workflow.configureWorkflow(
+				workflowDefinition = "Single Approver",
+				workflowResourceValue = "Custom Object 156806");
+		}
+
+		task ("Create entry (approved) via API") {
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customObjectField",
+				objectName = "CustomObject156806",
+				value = "Entry Approved");
+		}
+
+		task ("Define the entry as approved") {
+			Navigator.openURL();
+
+			ObjectAdmin.openMyWorkflowTasks();
+
+			Navigator.gotoNavTab(navTab = "Assigned to My Roles");
+
+			LexiconEntry.gotoEllipsisMenuItem(menuItem = "Assign to Me");
+
+			Workflow.confirmWorkflowAction();
+
+			LexiconEntry.gotoEllipsisMenuItem(menuItem = "Approve");
+
+			Workflow.confirmWorkflowAction();
+		}
+
+		task ("Create entry (pending) via API") {
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customObjectField",
+				objectName = "CustomObject156806",
+				value = "Entry Pending");
+		}
+
+		task ("Go to Custom Object entries and assert entry with filter") {
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject156806");
+
+			ObjectPortlet.viewEntry(entry = "Entry Approved");
+
+			ObjectPortlet.assertEntryNotPresent(entry = "Entry Pending");
+		}
+
+		task ("Change filter by Status") {
+			ObjectCustomViews.clickOnFilterOnEntries();
+
+			SelectFrame(locator1 = "ObjectCustomViews#VIEW_DROPDOWN_MENU_ON_MODAL_DEFAULT_SORT");
+
+			Type(
+				locator1 = "ObjectCustomViews#SEARCH_DROPDOWN_MENU_FILTER",
+				value1 = "Status");
+
+			Click(
+				key_optionName = "Status",
+				locator1 = "ObjectCustomViews#OPTIONS_DROPDOWN_MENU_FILTER");
+
+			ObjectCustomViews.optionsStatusOnEntryFilter(optionStatus = "Draft");
+
+			Check.checkNotVisible(locator1 = "ObjectCustomViews#TOGGLE_SWITCH_EXCLUDE_STATUS");
+
+			Click(locator1 = "ObjectCustomViews#BUTTON_EDIT_FILTER");
+		}
+
+		task ("Assert entry with new filter by Status") {
+			ObjectCustomViews.clickOnFilterOnEntries();
+
+			ObjectPortlet.viewEntry(entry = "Entry Pending");
+
+			ObjectPortlet.assertEntryNotPresent(entry = "Entry Approved");
+		}
+
+		task ("Change filter by Create Date") {
+			ObjectCustomViews.clickOnFilterOnEntries();
+
+			SelectFrame(locator1 = "ObjectCustomViews#VIEW_DROPDOWN_MENU_ON_MODAL_DEFAULT_SORT");
+
+			ObjectCustomViews.clickOnBackInFilter();
+
+			Type(
+				locator1 = "ObjectCustomViews#SEARCH_DROPDOWN_MENU_FILTER",
+				value1 = "Create Date");
+
+			ObjectCustomViews.defineDateCreateDateFilter(fromCreateDate = "01012022");
+		}
+
+		task ("Change filter by Modified Date") {
+			ObjectCustomViews.clickOnBackInFilter();
+
+			Type(
+				locator1 = "ObjectCustomViews#SEARCH_DROPDOWN_MENU_FILTER",
+				value1 = "Modified Date");
+
+			ObjectCustomViews.defineDateModifiedDateFilter(fromModifiedDate = "01012022");
+		}
```

Comment:
@andreefarias I see we're changing the filters there to `Created Date` and `Modified Date` but it doesn't look like the test is actually using these to filter the entries. What is the test covering here?